src/components/bulkConcepts/component/PreviewCard.jsx

```diff
@@ -82,6 +82,23 @@ const PreviewCard = ({
         remainingItems={answers.slice(NUMBER_OF_ITEMS_TO_DISPLAY).join(' ')}
       />;
     }
+    if (answers && answers.length > 0 && setMembers && setMembers.length > 0) {
```

Comment:
What happens when answers.length > 0 but setMembers.length = 0?

Reply:
Line 76 handles this scenario. But ideally what would happen is only answer mappings would be displayed

Reply:
My question is, if answers.length > 0 but setMembers.length = 0, does this mean you would not render answers?

Reply:
`if (answers && answers.length > 0 && setMembers && setMembers.length < 1)`
This condition would pass in this scenario, and hence the answers would be displayed. Would it be more satisfactory if I equated the code to something like
(answers && answers.length > 0 && setMembers && setMembers.length = 0?
 My understanding is it is similar to the code I currently have. Or am I missing something else?

Reply:
When you say that the condition would pass, which line are you talking about?

Reply:
Im talking about `line 76` in file `src/components/bulkConcepts/component/PreviewCard.jsx`

Reply:
My questions are in reference to line 85

Reply:
I have refactored the code to use `if else-if` statements. Would that cater for your concern?

Reply:
What do your team mates say?

Reply:
I had afew suggestions, is it possible to give it a final glance?

Reply:
> My question is, if answers.length > 0 but setMembers.length = 0, does this mean you would not render answers?



> `if (answers && answers.length > 0 && setMembers && setMembers.length < 1)`
> This condition would pass in this scenario, and hence the answers would be displayed. Would it be more satisfactory if I equated the code to something like
> answers && answers.length > 0 && setMembers && setMembers.length = 0?
> My understanding is it is similar to the code I currently have. Or am I missing something else?

I believe this situation is catered for in the scenario Paul outlines, @dkayiwa. A little clarity on what we might be missing perhaps?

Reply:
Why do you combine the check for setmembers and answers in the same if condition?

Reply:
Thank you for the clarity, I should effect the changes shortly

Reply:
I have refactored the code please have a look at it when you get some time